### DISCUSSION TITLE
V2.0 Public Release

### DISCUSSION THREAD

**Discussion by AdamC1228** [Erabior/RouteManager#98] [General]
V2.0 (V2.0.0.8) has been released and is considered the first stable 2.0 release. 

This is the general release discussion thread.

**Comment by AdamC1228** [comment]
@Erabior Unless you object I created a v2.0 branch for you to set as the default branch so the v2.0 readme will show by default.

**Reply by Erabior** [reply]
No objections

**Comment by Narflar** [comment]
is the work for this mod being used by the Game Devs for their planned Passenger AI?

**Reply by AdamC1228** [reply]
This project is independent and to my knowledge no game devs have in contact with anyone contributing to this project.
